vkbeautify/vkbeautify-tests.ts

```diff
@@ -0,0 +1,119 @@
+import VKBeautify from 'vkbeautify';
```

Comment:
the name space is not really required. modules can just export functions directly.